pyevmasm/__main__.py

```diff
@@ -18,9 +18,26 @@ def main():
                         help='Input file, default=stdin')
     parser.add_argument('-o', '--output', nargs='?', default=sys.stdout, type=argparse.FileType('w'),
                         help='Output file, default=stdout')
+    parser.add_argument('-f', '--fork', nargs=1, default=DEFAULT_FORK, type=str,
```

Comment:
```suggestion
    parser.add_argument('-f', '--fork', default=DEFAULT_FORK, type=str,
```

Reply:
I assume there is no reason to specify a list of forks. When `nargs=1` is not passed, `args.fork` will return a string instead of a single item list with a string.

Reply:
Ok thanks. I will change it back to '?' (see my comment above).